tools/http-fault-injector/sample-clients/java/storage-blobs/src/main/java/App.java

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import com.azure.core.util.BinaryData;
+import com.azure.core.util.Configuration;
+import com.azure.storage.blob.BlobClient;
+import com.azure.storage.blob.BlobClientBuilder;
+import com.azure.storage.common.policy.RequestRetryOptions;
+import com.azure.storage.common.policy.RetryPolicyType;
+
+import java.time.Duration;
+
+public class App {
+    public static void main(String[] args) {
+        // You must either add the .NET developer certiifcate to the Java cacerts keystore, or uncomment the following
```

Comment:
```suggestion
        // You must either add the .NET developer certificate to the Java cacerts keystore, or uncomment the following
```

Typo

Reply:
Copied and pasted this, will fix both locations